Source/Core/ScreenSpaceEventHandler.js

```diff
@@ -53,7 +53,7 @@ import ScreenSpaceEventType from './ScreenSpaceEventType.js';
         function listener(e) {
             callback(screenSpaceEventHandler, e);
         }
-        element.addEventListener(domType, listener, false);
+        element.addEventListener(domType, listener, { capture: false, passive: false });
```

Comment:
Sorry for the slow review here.  It looks like `options` objects are not supported by IE11 See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

We should probably detect IE11 here and pass boolean vs an object.

Reply:
@mramato, you are right, I see it in compatibility table below now. Is it ok if we check the browser by parsing `window.navigator.userAgent` here? Or should we add some advanced external dependency to do it such as [bowser](https://github.com/lancedikson/bowser)?

Reply:
Sorry, I should have provided more concrete feedback here.  Cesium has a `FeatureDetection` module, so you can import it and call `FeatureDetection.isInternetExplorer()` to check. Thanks!

Reply:
Updated PR.